python/cuml/cuml/preprocessing/ordinalencoder_mg.py

```diff
@@ -19,6 +19,8 @@ def _check_input_fit(self, X, is_categories=False):
 
         from cuml.dask.common.dask_arr_utils import to_dask_cudf
 
+        self._check_n_features(X, reset=True)
```

Comment:
This was a bug that was masked by the magical decorating from the base class. Fixed.

Note that `cuml.preprocessing.encoders` currently implements its own different reflection system, and will need to be migrated over in a followup.